src/main/java/com/kosherjava/zmanim/hebrewcalendar/TefilaRules.java

```diff
@@ -512,6 +512,49 @@ public boolean isMizmorLesodaRecited(JewishCalendar jewishCalendar) {
 	    return true;
 	}
 	
+	/**
+	 * Returns if <em>Selichos</em> is recited on the day in question. 
+	 * @param jewishCalendar  the Jewish calendar day.
+	 * @param isSefardi  whether this is the Sefardi or Ashkenazi minhag.
+	 * @return if <em>Selichos</em> is recited.
+	 * 
+	 */
+	public boolean isSelichosRecited(JewishCalendar jewishCalendar, boolean isSefardi) {
+		int holidayIndex = getYomTovIndex();
+
+		// All fast days except Tisha B'Av say Selichos even if they fall out on Shabbos (only Yom Kippur can)

```

Comment:
Sepharadim only have selichot on Yom Kippur & Gedalia.

Reply:
Are you saying that it is not said the rest of the Aseres Yimei Teshuva? It sounds like different opinions, because I've heard that they do say it.

Reply:
Sorry, this comment was meant just to be in reference to the other fast days.